ui/scripts/templates.js

```diff
@@ -1378,12 +1378,7 @@
                                     isextractable: {
                                         label: 'label.extractable.lower',
                                         isBoolean: true,
-                                        isEditable: function() {
-                                            if (isAdmin())
-                                                return true;
-                                            else
-                                                return false;
-                                        },
+                                        isEditable: true,
```

Comment:
Can you add a check to only allow editable option when it's either the root admin of when template account matches the logged in user account?

Reply:
The edit button is not shown for public templates when logged in with a different account, so the extractable checkbox is not displayed.